tests/unit/commands/init/test_cli.py

```diff
@@ -17,7 +17,12 @@
 
 
 class MockInitTemplates:
-    def __init__(self, no_interactive=False):
+    def __init__(self, no_interactive=False, auto_clone=True):
+        self._manifest_url = "https://raw.githubusercontent.com/sapessi/aws-sam-cli-app-templates/master/manifest.json"
```

Comment:
Will there be a PR out to aws-sam-cli-app-templates with the proposed changes to manifest.json?

Reply:
there is a pr already this already link here: https://github.com/aws/aws-sam-cli-app-templates/pull/90